setup.py

```diff
@@ -3,6 +3,9 @@
 import shutil
 import sys
 
+if sys.platform == "darwin":
+    os.environ.setdefault("MACOSX_DEPLOYMENT_TARGET", "10.9")
```

Comment:
I read up on this briefly because it's not mentioned in the PR text- seems to be a flag for indicating the oldest macos version this will work on

Reply:
I should have made note of it:
* It's present in the `Makefile` https://github.com/pcdshub/whatrecord/blob/cdeb432b81c7f1f9e0ee9841a50190126a3bba23/Makefile#L19
* I usually forget the `Makefile` exists and get confused when the build fails, so I added it here
* This is required for clang to actually build whatrecord with cython on macOS - it's a bit of a mess

I'm hoping to remove cython entirely from this project and relocate it, but that's probably not a today-thing...